docs/victory-axis/docs.md

```diff
@@ -22,6 +22,15 @@ The `tickCount` prop specifies approximately how many ticks should be drawn on t
 
 *default:* `tickCount={5}`
 
+### fixLabelOverlap
+
+When true, axis tick labels and their associated ticks
+and grid lines will be removed until the total calculated length of the remaining labels is less
+than the length of the axis. Labels at approximately even intervals from the original array of labels.
```

Comment:
I think this is just a fragment, not a sentence... "Labels at approximately even intervals from the original array of labels."